src/lambda.ts

```diff
@@ -20,14 +20,23 @@ type PublicationInfo = {
  * The AWS Lambda handler function.
  * Runs a series of checks against today's Kindle publication, then sends an email
  */
-export async function handler(): Promise<SendEmailResponse> {
+export async function handler(): Promise<SendEmailResponse | string> {
+    let currentHour = new Date().getHours();
+
+    if (shouldRun(currentHour)) return run();
+    else return Promise.resolve(`Not running because hour is ${currentHour}`)
+}
+
+let shouldRun = (currentHour: number): boolean => config.RunHours.indexOf(currentHour) >= 0;
```

Comment:
you can use `config.RunHours.includes(currentHour)`

Reply:
I tried `include` and it wasn't an available method locally...